tuf/keys.py

```diff
@@ -93,6 +94,15 @@
 except ImportError: # pragma: no cover
   pass
 
+# Import the pyca/Cryptography library so that RSA and generate-purpose
+# cryptography primitives are supported. 
+try:
+  import cryptography
```

Comment:
AFAICT `cryptography` isn't used as a library in this module; same for `Crypto` above. Sub-modules of both are imported in the respective `tuf.*_keys` modules.